Documentation/benchmarks/etcd-3-watch-memory-benchmark.md

```diff
@@ -0,0 +1,72 @@
+# Watch Memory Usage Benchmark
+
+*NOTE*: The watch features are still being actively developed and we expect the memory usage to change over time. However we do expect that memory usage should not be significantly increased from the figures below.
```

Comment:
*Note*: The *watch* features are under active development, and their memory usage may change as that development progresses. We do not expect it to significantly increase beyond the figures stated below.